ProjectSettings/InputManager.asset

```diff
@@ -6,36 +6,36 @@ InputManager:
   serializedVersion: 2
   m_Axes:
   - serializedVersion: 3
-    m_Name: Horizontal
+    m_Name: LeftVRGripAxis
```

Comment:
Are our controllers reliant on these input additions?

Or do they use the direct ButtonID inputs?
https://docs.unity3d.com/Manual/OculusControllers.html